### PR TITLE
Login page changes

### PR DESCRIPTION
**Pull request by jdtoombs**
Updated to match latest UI/UX 
newlogin

### CODE REVIEW

File: .github/workflows/api-dotnetcore.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
         run: dotnet restore
         working-directory: ${{env.working-directory}}
       - name: Install coverlet for code coverage
-        run: dotnet tool install -g coverlet.console
+        run: dotnet tool install -g coverlet.console --ignore-failed-sources
```
Comment (Fosol) [inline .github/workflows/api-dotnetcore.yml:26]: Is this to fix the coverlet GitHub action issue?  The error makes me wonder if we should be using a specific version, or a release version at least.

**Review by Fosol — APPROVED** [review]

File: .github/workflows/api-dotnetcore.yml (hunk 1 above)
Comment (jdtoombs) [reply .github/workflows/api-dotnetcore.yml:26]: It was an attempt, no luck with it though. And yeah me too, I am going to try a few other things here.

File: .github/workflows/api-dotnetcore.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ jobs:
         run: dotnet restore
         working-directory: ${{env.working-directory}}
       - name: Install coverlet for code coverage
-        run: dotnet tool install -g coverlet.console
+        run: dotnet tool install --global coverlet.console --version 1.7.2
```
Comment (devinleighsmith) [inline .github/workflows/api-dotnetcore.yml:26]: is this a fix for the .net tests?

File: frontend/src/features/account/Login.scss [hunk 3]
```diff
@@ -1,44 +1,49 @@
...
   p {
-    font-size: 16px;
+    font-size: 12px;
     color: $text-color;
     font-family: 'BCSans', Fallback, sans-serif;
     line-height: 130%;
-    width: 75%;
+    margin-left: 30px;
+    margin-right: 30px;
+    margin-bottom: -10px;
```
Comment (devinleighsmith) [inline frontend/src/features/account/Login.scss:32]: What's going on here that we need a negative margin?

File: frontend/src/features/account/Login.tsx [hunk 4]
```diff
@@ -29,13 +29,23 @@ const Login = () => {
...
-      <h1>Search and manage government properties</h1>
+      <h1>Search and visualize government property information</h1>
       <Row>
         <Col xs={1} md={3} />
         <Col xs={16} md={6} className="block">
+          <h6>PIMS enables you to search properties owned by the Government of British Columbia</h6>
+          <p>
+            The data provided can assist your agency in making informed, timely, and strategic
+            decisions on the optimal use of real property assets on behalf of the people and
+            priorites of British Columbia.
```
Comment (devinleighsmith) [inline frontend/src/features/account/Login.tsx:40]: typo.

**Review by devinleighsmith — APPROVED** [review]

File: .github/workflows/api-dotnetcore.yml (hunk 2 above)
Comment (jdtoombs) [reply .github/workflows/api-dotnetcore.yml:26]: it was an attempt but did not work

File: frontend/src/features/account/Login.scss (hunk 3 above)
Comment (jdtoombs) [reply frontend/src/features/account/Login.scss:32]: fixed in next pr about to put up

**Pull request merged**